### PR TITLE
My first pull request!!

### PR DESCRIPTION
**Pull request by Ade179**
### In this milestone 

- I added linters configuration to my project
- I used  correct github workflow
- I documented my work in a professional way

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+Twitter: @Juwon_Adesanya
+LinkedIn: Juwon Adesanya
+
+
+
+Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This was issued by microverse
```
Comment (GzimAsani) [inline README.md:31]: - [ ] Update your readme file to make it more professiona, use this [Template](https://github.com/microverseinc/readme-template)

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @Ade179        :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the html and css :+1: 
 
 ## Required Changes 

- [ ] Please add a better descriptive description in your pull request
- [ ] You should add linters in your project, take a step back and check this [document](https://github.com/microverseinc/linters-config/tree/master/html-css) on how to add linters for your project
- [ ] There needs to be a .gitignore file listing test.md file and node_modules directory
- [ ] _Check the comments below :arrow_double_down:_



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ade179** [comment]
Ok..thank you. I will work on it

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Ade179               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**